src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/ScopedValue.java

```diff
@@ -43,71 +42,109 @@
 import sun.security.action.GetPropertyAction;
 
 /**
- * Represents a value that is local to a <em>scope</em>. It is a per-thread value
- * that allows context to be set in a caller and read by callees. The <em>scope</em> is
- * the set of methods that the caller directly invokes, and any methods invoked
- * transitively. Scoped values also provide a way to share immutable data across
- * threads.
+ * A value that is set once and is then available for reading for a bounded period of
+ * execution by a thread. A {@code ScopedValue} allows for safely and efficiently sharing
+ * data for a bounded period of execution without using method arguments.
  *
```

Comment:
This reads oddly?
```suggestion
 * data for a bounded period of execution without using passing them as method arguments.
 *
```

Reply:
Yes, and "passing the data" might be a better again.